build/Build.cs

```diff
@@ -329,6 +391,16 @@ public Build()
                                     }
                                 }
 
+                                foreach (var flavour in MigratedCalamariFlavours.Flavours)
+                                {
+                                    packageReferences.Add(new BuildPackageReference
+                                    {
+                                        Name = flavour,
+                                        Version = NugetVersion.Value,
+                                        PackagePath = $"{ArtifactsDirectory}\\{flavour}.zip"
```

Comment:
Is this a typo or is it supposed to be `\\`?

Reply:
Supposed to be \ escapes the next character so \\ is required to use a \. Is this correct for Mac aswell?

Reply:
Yeah duh. Ignore this, dumb comment in the morning 🐵 
Wasn't apparent to me that we had to escape when creating a path as we generally use forwardslashes instead of a backslash